lib/gcp/cloudfunctions.js

```diff
@@ -23,7 +23,7 @@ function _functionsOpLogReject(func, type, err) {
 
 function _generateUploadUrl(projectId, location) {
   var parent = 'projects/' + projectId + '/locations/' + location;
-  return api.request('POST', '/v1/' + parent + '/functions:generateUploadUrl', {
+  return api.request('POST', API_VERSION + parent + '/functions:generateUploadUrl', {
```

Comment:
this won't work. you're missing the slashes.

Reply:
😲sorry!! thanks for catching it